turbinia/notify.py

```diff
@@ -0,0 +1,57 @@
+import logging
+from turbinia import config
+
+#Sends notfications via email
+def sendmail(message):
```

Comment:
Could you add an optional parameter here for an alternate username or email address?  If it's a username, then you could have a DEFAULT_EMAIL_DOMAIN (or something like that) in the config, and then when the email gets sent from the server, it can use the `requester`, which is set to the username of the person making the request.  That way we can email the user who made the processing request.